src/pages/ExpensePage.jsx

```diff
@@ -1,5 +1,37 @@
-function Expense() {
-  return <h2>Expense</h2>;
-}
+import Table from "../components/Table";
+
+const rows = [
+  { id: 1, name: "ExpenseGroupName", description: "Description" },
+  { id: 2, name: "ExpenseGroupName22222", description: "Description222" },
+];
+const columns = [
+  {
+    field: "id",
+    headerName: "ID",
+    width: 100,
+  },
+  {
+    field: "name",
+    headerName: "Name",
+    width: 300,
+    editable: true,
+  },
+  {
+    field: "description",
+    headerName: "Description",
+    width: 350,
+    editable: true,
+  },
+];
+
+const Expense = () => {
+  return (
+    <h2>
+      Expense
+      <br />
+      <Table valueRows={rows} valueColumns={columns} />
```

Comment:
Here the same as we said in the Table component, you can just simply call them rows and columns instead of valueRows and valueColumns